subsystems/win32/csrsrv/server.c

```diff
@@ -153,12 +153,11 @@ CsrLoadServerDll(IN PCHAR DllString,
             RtlInitUnicodeString(&ErrorString, L"Default Load Path");
 
             /* Send a hard error */
-            NtRaiseHardError(Status,
-                             2,
-                             3,
-                             Parameters,
-                             OptionOk,
-                             &Response);
+            Status2 = NtRaiseHardError(Status, 2, 3, Parameters, OptionOk, &Response);
```

Comment:
Please restore the formatting (and we don't really need the returned Status code, see below).

Reply:
Reverted.